src/Fields.jl

```diff
@@ -47,3 +47,59 @@ function canonical_unit(x::FieldElement)
   iszero(x) && return one(x)
   return x
 end
+
+@doc raw"""
+   evaluation_points(K::Field, n::Int) -> Vector{FieldElem}
+   evaluation_points(K::AbstractAlgebra.Generic.RationalFunctionField, n::Int) -> Vector{RationalFunctionFieldElem}
+
+If $K$ contains at least $n$ elements, this function returns an $n$-element vector $v$ with pairwise different entries from the given field $K$. Otherwise it returns an empty vector. 
+"""
+function evaluation_points(K::Field, n::Int)
+   if n <= 0
+      error("n must be positive!")
+   end
+   if is_finite(K)
+      v = elem_type(K)[]
+      if (order(K) < n)
+         return v
+      else
+         for a in K
+            push!(v, a)
+            if length(v) == n
+               break
+            end
+         end
+      end
+   else
```

Comment:
```suggestion
   else
      @assert characteristic(K) == 0 "This should not happen. Please open a bug report."
```
To be on the safe side.